### DISCUSSION TITLE
I am unable to connect to Athena from superset via jdbc driver.

### DISCUSSION THREAD

**Discussion by knadigatla** [apache/superset#15998] [Q&A / Help]
Hello All,

I am trying to connect to athena using PyAthenaJDBC module. I used docker in local to run the superset, and I added the required pip module dependency. But I am facing a java error(as PyAthenaJDBC is uses jdbc driver and it needs java installed) ```ERROR: No JVM shared library file (libjvm.so) found. Try setting up the JAVA_HOME environment variable properly.```

Can you help me me install the java and set the JAVA_HOME in superset container.??

**Comment by nytai** [comment]
You'll have to modify the [dockerfile](https://github.com/apache/superset/blob/46188c1011fdf91876cd97215d49068f22368fd8/Dockerfile) to include java

**Reply by knadigatla** [reply]
In my local it may work fine, but I want to deploy superset to Kubernetes using EKS. Where should I update then? Should I custom build the docker images?
